src/engine/fileOps.cpp

```diff
@@ -307,12 +307,12 @@ bool DivEngine::loadDMF(unsigned char* file, size_t len) {
         if (!ins->mode) {
           ins->type=DIV_INS_AY;
         }
-        ins->std.dutyMacroHeight=31;
-        ins->std.waveMacroHeight=7;
+        ins->std.dutyMacro.height=31;
```

Comment:
Those height variables are no longer used.

Reply:
I was planned for macro execute flag with using this, but finally cancelled.